src/modules/layout/containers/GeneralHeader.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
```

Comment:
why did you move GeneralHeader to the containers package? It does not connect to the store nor does it use withFetcher or translations

Reply:
Okay, but then LocationHeader and GeneralHeader should also be containers because Header is connecting to the state

Reply:
Hmm.. yap... Do you think, we need a better definition of when to call a ReactComponent a container and when to call it a component? Or does this distinction still make sense in our project at all? 🤔
At least I like, that we have some distinction between complex ReactComponents, that can modify the state, vs dumb / just visual components...